code/hdb/hdbstandard.q

```diff
@@ -4,4 +4,28 @@ reload:{
 	system"l ."}
 
 // Get the relevant HDB attributes
-.proc.getattributes:{`date`tables!(@[value;`date;`date$()];tables[])}
+\d .proc
+getattributes:{
+	timecolumns:1!except[(::),gettimecolumns each t:tables[`.];(::)];
+	attrtable:`date`tables`procname!(
+		/ date reporting makes some assumptions
+		@[value;`date;`date$()] union first[d]+til 1+last deltas d:exec(min;max)@\:distinct`date$last each raze value each timecolumns from timecolumns;
+		tables[];
+		.proc.procname);
+	attrtable[`dataaccess]:`segid`tablename!(first .ds.segmentid;(key timecolumns)!select wc,timecolumns from update wc:{""}'[i],timecolumns from timecolumns);
```

Comment:
why is this added to the `attrtable` dictionary separately when the other three keys are all added together above? (I prefer this one as it has the key on the same line as the value so it's clearer IMO, but would be best to keep things consistent - ideally change the other keys to match this)